org.adempiere.base/src/org/adempiere/impexp/GridTabCSVExporter.java

```diff
@@ -301,6 +316,19 @@ else if (sValue != null && ! (Boolean) sValue)
 		} catch (IOException e) {
 			throw new AdempiereException(e);
 		} finally {
+			gridTab.setCurrentRow(currentRow);
+			
+			// reload/query for all child tab to avoid any side effect
+			for (GridTab childTab:childs){
+				if (!childTab.isLoadComplete()){
+					childTab.initTab(false);
+				}
+				childTab.query(false, 0, 0);
+			}

```

Comment:
Don't think we need to re-query all child here. It is just export and very unlikely to cause any side effect.